.github/workflows/ci.yml

```diff
@@ -65,10 +80,17 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - uses: wyvox/action-setup-pnpm@v3
+      - uses: pnpm/action-setup@v4
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
-          pnpm-version: ${{ env.PNPM_VERSION }}
+          cache: pnpm
+      - name: Install Dependencies
+        run: pnpm install --frozen-lockfile
+      # This has been really annoying
+      # due to injected dependencies :(
+      - name: 'Re-sync injected dependencies'
+        run: pnpm install --frozen-lockfile --force
       - name: Run Tests
-        run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }}
+        run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }} --skip-cleanup
```

Comment:
you can't rely on the node_modules location here

you need to change this to `pnpm ember try:one`

Reply:
We better fix this in the blueprint in that case